### PR TITLE
Create a new integration category (RPC Node Providers) and add Omnia Protocol

### PR DESCRIPTION
**Pull request by dumitrucatalin**

### CODE REVIEW

File: docs/integrate/rpc-node-providers/omnia.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+sidebar_position: 1
+---
+
+## Introduction
+
+[OMNIA Protocol](https://omniatech.io) is a blockchain infrastructure provider that delivers optimal performance without sacrificing privacy, security or compliance.
+
+## Privacy and Consumer Protection
+
+Transactions can be tracked even before they get mined, compromising privacy in the face of malicious actors who track your behavior. OMNIA prevents this by obfuscating your off-chain data, thus safeguarding you against scams, hacks, and front-running attacks. All the data and metadata remain private to the users. No third party is able to access, analyze or track it. OMNIA leverages different technologies and approaches to guarantee the privacy of their users, from front-running protection and private mempools, to obfuscation and random dispatching.
```
Comment (Karkunow) [inline docs/integrate/rpc-node-providers/omnia.md:12]: We're proof-of-stake blockchain - therefore we need to change "mined" here :)

**Review by Karkunow — CHANGES_REQUESTED** [review]

File: docs/integrate/rpc-node-providers/omnia.md (hunk 1 above)
Comment (dumitrucatalin) [reply docs/integrate/rpc-node-providers/omnia.md]: Got it, I will fix this.

**Review by Karkunow — APPROVED** [review]

**Review by Karkunow — APPROVED** [review]

**Comment by Karkunow** [comment]
@dumitrucatalin there is some lines with too many chars in them. Please, take a look!

**Comment by dumitrucatalin** [comment]
Thx @Karkunow, just fixed it. My local linter didn't find it last time.

**Pull request merged**